tests/plugins/test_check_and_set_rebuild.py

```diff
@@ -118,7 +118,7 @@ def test_check_is_not_rebuild(self, namespace, monkeypatch):
         build_json["metadata"].update(namespace_dict)
         monkeypatch.setenv("BUILD", json.dumps(build_json))
         runner.run()
-        assert workflow.prebuild_results[CheckAndSetRebuildPlugin.key] == False
+        assert not workflow.prebuild_results[CheckAndSetRebuildPlugin.key]
```

Comment:
I think we want `is False` here, to verify it's actually the Boolean value rather than some other value that can be coerced to it.